Mathlib/Algebra/Ring/Basic.lean

```diff
@@ -60,37 +60,40 @@ end AddHomClass
 namespace AddMonoidHom
 
 /-- Left multiplication by an element of a (semi)ring is an `AddMonoidHom` -/
-def mulLeft [NonUnitalNonAssocSemiring R] (r : R) :
-    R →+ R where
-  toFun := (· * ·) r
+def mulLeft [NonUnitalNonAssocSemiring R] (r : R) : R →+ R where
+  toFun := (r * ·)
   map_zero' := mul_zero r
   map_add' := mul_add r
 #align add_monoid_hom.mul_left AddMonoidHom.mulLeft
 
 @[simp]
-theorem coe_mul_left [NonUnitalNonAssocSemiring R] (r : R) :
-    (mulLeft r : R → R) = HMul.hMul r :=
+theorem coe_mulLeft [NonUnitalNonAssocSemiring R] (r : R) :
+    (mulLeft r : R → R) = (r * ·) :=
```

Comment:
I'd be more inclined not to change this one. Somebody might want to write `rw [<- coe_mulLeft]` and that becomes harder with the new statement.

Reply:
Yeah, that's what I added `mulLeft_apply` for. In fact, your PR #2725 introduced this change relative to mathlib3. I think it's better to be consistent with mathlib3 as well as with `coe_mulRight`.